CHANGELOG.md

```diff
@@ -4,7 +4,15 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
-## Unreleased
+## 2.6.0 (2022-02-16)
+This release adds major new features since the 2.5.2 release, among
+them:
+
+* Compression in continuous aggregates
+* Experimental support for timezones in continuous aggregates
+
+It also includes several bug fixes. We deem this release to be moderate
+priority for upgrading.
 
 **Features**
 * #3768 Allow ALTER TABLE ADD COLUMN with DEFAULT on compressed hypertable
```

Comment:
Do we have doc updates for these minor feature additions? @NunoFilipeSantos

Reply:
Or at least we should...